src/osd/PGLog.cc

```diff
@@ -276,6 +276,7 @@ void PGLog::proc_replica_log(
     divergent,
     oinfo,
     olog.get_can_rollback_to(),
+    olog.get_can_rollback_to(),
```

Comment:
since this is used after rewind_from_head() it looks like we could run into the same issue here

Reply:
yeah, I was hoping to fix that in a separate PR.

Reply:
sounds good